contracts/ConsumerHost.sol

```diff
@@ -69,7 +69,7 @@ contract ConsumerHost is Initializable, OwnableUpgradeable, IConsumer, ERC165 {
     event Withdraw(address consumer, uint256 amount, uint256 balance);
 
     /// @notice Emitted when consumer pay for open a state channel
-    event Paid(uint256 channelId, address consumer, address caller, uint256 amount, uint256 balance);
+    event Paid(uint256 channelId, address consumer, address caller, uint256 amount, uint256 balance, uint256 fee);
```

Comment:
This event not used by network subquery project, just confirm only consumer host service need to resynced the events

Reply:
y, consumer host service need update this event.